src/lib/progress-spinner/progress-spinner.spec.ts

```diff
@@ -108,6 +108,29 @@ describe('MdProgressSpinner', () => {
     expect(progressElement.componentInstance.interdeterminateInterval).toBeFalsy();
   });
 
+  it('should allow a custom stoke width', () => {
+    let fixture = TestBed.createComponent(BasicProgressSpinner);
+    let pathElement = fixture.nativeElement.querySelector('path') as HTMLElement;
+
+    fixture.detectChanges();
+
+    expect(parseInt(pathElement.style.strokeWidth))
```

Comment:
There's no need to parse the string to an int here. You can just expect it to be `'10'`.